xbmc/video/Teletext.cpp

```diff
@@ -1285,7 +1285,6 @@ void CTeletextDecoder::RenderPage()
         else
         {
           SetPosX(33+i);
-          m_RenderInfo.PageChar[32+i] = m_RenderInfo.PageChar[32+i];
```

Comment:
Are we sure that this is the right fix?

Reply:
indeed it's not obvious to me what's the purpose of the line in its original state. just did a runtime-test without it and haven't seen regression on the teletext. what could be behind it, you have an idea? @ksooo

Reply:
Sorry, no idea. But I would like to get an idea, what the real intention of the writer of that code was instead of "blindly" removing this line. If we now delete it we delete any trace that obviously here should be done more than nothing.

Reply:
OMG, that code is 11 years old... but the person who comitted it is still around, @AlwinEsch would do you say? Do you remember your intentions?

Reply:
I notice now that this issue was posted before but was closed for not use the format. https://github.com/xbmc/xbmc/issues/15361

Reply:
this is the corresponding compiler warning. let's just wait for Alwin's response. then we can go on ;)

Reply:
It is from the time I stole the code, unfortunately it is still there in the original.

I think it was about bringing in something for flashing, which was probably never finished.

In any case, removing it makes sense.

Reply:
thanks @AlwinEsch